docs/conf.py

```diff
@@ -1,10 +1,43 @@
-import sys
 import os
+import pathlib
+import sys
+
+import furo
+import furo.navigation
 
 sys.path.append('./')
 from custom_conf import *
 sys.path.append('.sphinx/')
 from build_requirements import *
+sys.path.insert(0, str(pathlib.Path(__file__).parent.parent))
+
+# Furo patch to get local TOC to show in sidebar (as sphinx-rtd-theme did)
+# See https://github.com/pradyunsg/furo/blob/490527b2aef00b1198770c3389a1979911ee1fcb/src/furo/__init__.py#L115-L128
+
+_old_compute_navigation_tree = furo._compute_navigation_tree
```

Comment:
Should we do these patches in `custom_conf.py` so we can leave `conf.py` untouched? What do you think?

If we decide to do that, let's just not lint `conf.py` at all, to avoid all the lint per-file ignores.

Reply:
OK, I will move these patches to `custom_conf.py` and test again.